.gitignore

```diff
@@ -1,5 +1,3 @@
 __pycache__
 .env
-spack.lock
-.spack-env
```

Comment:
It'd be nice to keep these in the `.gitignore` for other folks who might be using Spack.